pos_warning_exiting/static/src/js/gui.js

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define("pos_warning_exiting.gui", function (require){
+
+    "use strict";
+    var core = require("web.core");
+    var rpc = require('web.rpc');
+    var utils = require('web.utils');
+    var gui = require("point_of_sale.gui");
+
+    var _t = core._t;
+    var round_pr = utils.round_precision;
+
+
+    var _super_close_ = gui.Gui.prototype.close;
+
+    gui.Gui.prototype.close = function(){
```

Comment:
Is there a reason for using prototype instead of `gui.Gui.include`? :thinking:

Reply:
What is the other way ? 
I mean : 

local ``close`` calls  local ``check_connection_and_close`` that calls super of ``close``. 
Using prototype seems necessary.

Reply:
Oh yes, I see that now 👍🏻